### PR TITLE
Jon Janulewicz html-css response

### PR DESCRIPTION
**Pull request by modularjon**
Answer all questions.

Comfort: 4
Clarity: 4

Used MDN, css-tricks.com, and class notes to answer questions.

### CODE REVIEW

**Comment by modularjon** [comment]
OH NO TRICK LAST QUESTION. I meant 25px.

File: diagnostic.md [hunk 1]
```diff
@@ -31,7 +31,10 @@ When we say that the web is a service, what does that mean? Explain the
 interaction in terms of clients and servers.
 
 <!-- your answer starts here -->
-
+The web is a service for users, who initiate client requests to a server. The
```
Comment (berziiii) [inline diagnostic.md:34]: Correct.

File: diagnostic.md [hunk 2]
```diff
@@ -41,7 +44,21 @@ format described in the instructions). The page should have a title in the head
...
+    <link href="./stylesheet.css" rel="stylesheet" type="text/css">
+  </head>
+  <body>
+    <ul>
+      <li>MYTHINGS</li>
+      <li>MYSTUFF</li>
+      <li>MYITEMS</li>
+    </ul>
+  </body>
+</html>
```
Comment (berziiii) [inline diagnostic.md:60]: Correct.

File: diagnostic.md [hunk 3]
```diff
@@ -55,6 +72,27 @@ How would we reference each of the following categories of elements?
...
+```CSS
+.big {
+
+}
+
+#contentPane {
+
+}
+
+.important .red {
```
Comment (berziiii) [inline diagnostic.md:88]: `.important.red` would apply the styling to elements that have BOTH of these classes. With the space, you're saying apply the styles to both the `.important` and `.red` class. Big difference.

File: diagnostic.md [hunk 4]
```diff
@@ -99,7 +137,8 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+specialDiv's content would have a font-family of cursive and a text color of
```
Comment (berziiii) [inline diagnostic.md:140]: Correct.

File: diagnostic.md [hunk 5]
```diff
@@ -110,7 +149,7 @@ Order the following parts of the _box model_ from **'innermost'** to
 content, border, margin, padding
 
 <!-- your answer starts here -->
-
+(inner) content > padding > border > margin (outer)
```
Comment (berziiii) [inline diagnostic.md:152]: Correct.

File: diagnostic.md [hunk 6]
```diff
@@ -184,5 +223,5 @@ with the class `.stretch` be at a _screen width_ of 550px?
 ```
 
 <!-- your answer starts here -->
-
+30px
```
Comment (berziiii) [inline diagnostic.md:226]: At a screen width of 550px, the `@media(min-width= 400px)` would apply. `.stretch` is 25% of width so 25% \* 550 = 137.5px.

File: diagnostic.md [hunk 7]
```diff
@@ -157,7 +196,7 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+divOne above divFour on right, rest above eachother sequentially on left.
```
Comment (berziiii) [inline diagnostic.md:199]: 2,3,1
4,5

Would be the layout.

**Comment by berziiii** [comment]
4 / 7. Small errors but ones that came come back and make your life a living hell with debugging html/css.

**Pull request closed without merging**